src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -161,8 +176,18 @@ public void acceptAsLinked() {
 
     public void open() {
         try {
+            Optional<Path> resolvedPath = FileHelper.expandFilename(
```

Comment:
The old code also opens websites, which leads to problems with this new code. I think a `isLink` check at the appropiate place suffices.

Reply:
see above